chapter_preliminaries/probability.md

```diff
@@ -354,54 +808,235 @@ $$P(H = 1| D_1 = 1, D_2 = 1)
-
-We also sharpened our language and tools for statistical modeling. In the process of that we learned about conditional probabilities and about one of the most important equations in statistics---Bayes' rule. It is an effective tool for decoupling information conveyed by data through a likelihood term $P(B|A)$ that addresses how well observations $B$ match a choice of parameters $A$, and a prior probability $P(A)$ which governs how plausible a particular choice of $A$ was in the first place. In particular, we saw how this affects our ability to diagnose diseases, based on the efficacy of the test *and* the prevalence of the disease itself (i.e., our prior). We strongly recommend that you play with the equations to see how the analysis changes if you change the prior. 
-
-Lastly, we introduced a first set of nontrivial questions about the effect of a specific probability distribution, namely expectations and variances. While there are many more than just linear and quadratic expectations for a probability distribution, these two already provide a good deal of knowledge about the possible behavior of the distribution. For instance, [Chebyshev's Inequality](https://en.wikipedia.org/wiki/Chebyshev%27s_inequality) states that $P(|X - \mu| \geq k \sigma) \leq 1/k^2$, where $\mu$ is the expectation, $\sigma^2$ is the variance of the distribution and $k > 1$ is a confidence parameter of our choosing. It tells us that draws from a distribution lie with at least 50% probability within a $2 \sigma$ interval of the expectation. 
+Returning to our investment example,
+we can now compute the variance of the investment. 
+It is given by $0.5 \cdot 0 + 0.4 \cdot 2^2 + 0.1 \cdot 10^2 - 1.8^2 = 8.36$. 
+For all intents and purposes this is a risky investment. 
+Note that by mathematical convention mean and variance 
+are often referenced as $\mu$ and $\sigma^2$.
+This is particularly common whenever we use it 
+to parametrize a Gaussian distribution. 
+
+In the same way as we introduced expectations 
+and variance for *scalar* random variables, 
+we can do so for vector-valued ones. 
+Expectations are easy, since we can apply them element-wise. 
```

Comment:
element-wise -> elementwise (checked with you earlier and it's consistently used in the book)